src/tmb.cpp

```diff
@@ -632,21 +637,35 @@ Type objective_function<Type>::operator() ()
   // likelihood for ANC testing observations
 
   vector<Type> anc_clients_obs_t2((A_anc_clients_t2 * anc_clients_t2) * exp(offset_anc_clients_t2));
-  val -= dpois(x_anc_clients_t2, anc_clients_obs_t2, true).sum();
-	       
+  // val -= dpois(x_anc_clients_t2, anc_clients_obs_t2, true).sum();
```

Comment:
This all looks exactly right to me. I think you can go ahead and remove these commented out lines.